README.md

```diff
@@ -0,0 +1,27 @@
+# Half-Life SDK for Xash3D
+
+Half-Life SDK for Xash3D with some fixes.
+
+## How to build
+
+### Windows
+
+TODO
+
+### Linux
+
+TODO
+
+### OS X
+
+TODO
+
+### FreeBSD
+
+```
+    cd dlls
+    gmake CXX=clang++ CC=clang
```

Comment:
>There's no even such environment variable by default.

`
cat /usr/share/mk/sys.mk | grep CC
cat /etc/make.conf | grep CC
`
Good practice: gmake CC=${CC}
Because it's maybe changed into /etc/make.conf.
Also, i can upload pmake-compatible makefiles specially for *BSD, if needed.
Just *BSD uses different implementations of pmake by default, not gmake.